aviary/wren/utils.py

```diff
@@ -382,17 +386,21 @@ def count_params(aflow_label: str) -> int:
 
     num_params += cry_param_dict[pearson[0]]
 
-    subst = r"1\g<1>"
-    for wyk in wyks:
-        wyk = re.sub(r"((?<![0-9])[A-z])", subst, wyk)
-        sep_el_wyks = ["".join(g) for _, g in groupby(wyk, str.isalpha)]
+    for wyk_letters_per_elem in wyks:
+        # normalize Wyckoff letters to start with 1 if missing digit
+        wyk_letters_per_elem = re.sub(
+            r"((?<![0-9])[A-z])", r"1\g<1>", wyk_letters_per_elem
+        )
+        sep_el_wyks = [
+            "".join(g) for _, g in groupby(wyk_letters_per_elem, str.isalpha)
+        ]
         try:
```

Comment:
maybe this try except can go honestly, think it was more for debugging that production

Reply:
@janosh thoughts?

Reply:
sure, was considering removing it as well